kolibri/plugins/coach/assets/src/constants.js

```diff
@@ -4,6 +4,8 @@ const PageNames = {
   RECENT: 'RECENT',
   TOPICS: 'TOPICS',
   EXAMS: 'EXAMS',
+  CREATE_EXAM: 'CREATE_EXAM',
```

Comment:
should probably include all these new constants in a new file, like `examConstants`

see pattern here https://github.com/indirectlylit/kolibri/blob/7f78076a9846dc2e8747214b4f9be71373c497a8/kolibri/plugins/coach/assets/src/reportConstants.js